packages/gateway/tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "skipLibCheck": true,
     "baseUrl": "./src",
     "paths": {
-      "@gateway/*": ["./*"]
+      "@gateway/*": ["./*"],
+      "@events/*": ["../../events/src/*"]
```

Comment:
This is a little nasty – it is because gateway imports the `appRouter` from events

```ts
import type { AppRouter } from '@opencrvs/events/src/router'
```

which again imports modules inside Events with the `@events/...` paths.

```sh
❯ yarn test:compilation
yarn run v1.22.4
$ tsc --noEmit
../events/src/service/events.ts:12:27 - error TS2307: Cannot find module '@events/storage/mongodb' or its corresponding type declarations.

12 import { getClient } from '@events/storage/mongodb'
                             ~~~~~~~~~~~~~~~~~~~~~~~~~

```